api/tests/unit/organisations/test_unit_organisation_views.py

```diff
@@ -1340,3 +1340,40 @@ def test_list_my_groups(organisation, api_client):
         "id": user_permission_group_1.id,
         "name": user_permission_group_1.name,
     }
+
+
+def test_when_subscription_is_cancelled_then_remove_all_but_the_first_user(
+    staff_client: APIClient,
+    admin_client: APIClient,
```

Comment:
I don't think we need the `admin_client` here?

Reply:
Yep!